terraform/main.tf

```diff
@@ -301,29 +258,24 @@ output "dns_records" {
   value      = [
     "                  ${digitalocean_domain.ipfs-live-streaming.name} = ${digitalocean_domain.ipfs-live-streaming.ip_address}",
     "      ${digitalocean_record.rtmp-server.fqdn} = ${digitalocean_record.rtmp-server.value}",
-    "      ${digitalocean_record.ipfs-server.fqdn} = ${digitalocean_record.ipfs-server.value}",
-    "     ${digitalocean_record.ipfs-server-gateway.fqdn} = ${digitalocean_record.ipfs-server-gateway.value}",
+    "      ${digitalocean_record.rtmp-server.fqdn} = ${digitalocean_record.rtmp-server.value}",
```

Comment:
You are collapsing both servers into `rtmp-server`, but it still has an ipfs gateway, no?